packages/lib-ts/src/intents/Swap.ts

```diff
@@ -201,7 +147,29 @@ export class SwapBuilder extends IntentBuilder {
    */
   addTokenOutFromStringDecimal(token: Token, amount: string, recipient: Address): SwapBuilder {
     if (!token.hasChain(this.destinationChain)) throw new Error('Tokens out must be on the destination chain')
-    return this.addTokenOut(SwapTokenOut.fromStringDecimal(token, amount, recipient))
+    return this.addTokenOut(TokenOut.fromStringDecimal(token, amount, recipient))
+  }
+
+  /**
+   * Adds an output token from a slippage percentage string.
+   * @param tokenAmount - The token amount to receive (must be on destination chain)
+   * @param slippage - The slippage percentage as a decimal string
+   * @param recipient - The address to receive the tokens
+   * @returns This SwapBuilder instance for method chaining
+   */
+  addTokenOutFromSlippagePercent(tokenAmount: TokenAmount, slippage: string, recipient: Address): SwapBuilder {
```

Comment:
To be consistent with the naming
```suggestion
  addTokenOutFromSlippagePercentage(tokenAmount: TokenAmount, slippage: string, recipient: Address): SwapBuilder {
```